docs/pages/getting-started/4-connecting-crm.rst

```diff
@@ -0,0 +1,216 @@
+.. _connecting-crm:
+
+Getting Started on customer integration
+#######################################
+
+This getting started manual assumes a vendor sells services to its customers.
+The vendor manages the presence of those customers on the Nuts network through the Nuts registry.
+It's very likely the vendor has software to manage customer environments. We'll call it a *CRM* where the *customers* part relates to organizations and not people.
+This does not mean that a stand-alone installation isn't supported. In that case the vendor and organization are the same.
+
+The Nuts registry enables service discovery for organizations. The registry identifies organizations through their :ref:`DID <did>`.
+The DIDs are new and do not match the internal identifiers known to the CRM.
+This connection has to be made in order for the CRM to make adjustments to DID Documents and issue Verifiable Credentials.
+
+All APIs used in the following chapters are documented at the :ref:`API <nuts-node-api>` page.
```

Comment:
Maybe we should mention that integrators shouldn't store the DID Document itself in their software, but only the DID (because otherwise it can become stale when compared to the actual version on the network). So they don't need to store the result of the create operation, just its ID.